pkg/cmd/sync/sync.go

```diff
@@ -71,12 +67,23 @@ func (cmd *SyncCmd) Run(f *cmdutil.Factory) error {
 		}
 	}
 
+	originIds := make(map[string]contracts.AzionJsonDataOrigin)
+	for _, itemOrigin := range conf.Origin {
+		originIds[itemOrigin.Name] = contracts.AzionJsonDataOrigin{
+			OriginId:  itemOrigin.OriginId,
+			OriginKey: itemOrigin.OriginKey,
+			Name:      itemOrigin.Name,
+			Address:   itemOrigin.Address,
+		}
+	}
+
 	info := contracts.SyncOpts{
-		RuleIds: ruleIds,
-		Conf:    conf,
+		RuleIds:   ruleIds,
```

Comment:
You are not populating this structure with cacheIds

Reply:
good point :smile: